docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -231,6 +231,48 @@ docker run \
 
 The default Intel Quote Provider config is available in [Intel SGX Github repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/qcnl/linux/sgx_default_qcnl.conf).
 
+## Migrate from EPID Attestation to DCAP Attestation
+EPID attestation support has been discontinued in newer processors (Intel Xeon
+3rd generation onwards). All nodes using EPID attestation should migrate to DCAP
+Attestation.
```

Comment:
It is not just that, EPID attestation will be discontinued in 2025 and will no longer be available on any processors.